src/gl_fns.rs

```diff
@@ -1730,5 +1730,13 @@ impl Gl for GlFns {
             )
         }
     }
+
+    fn alias_point_size_range(&self) -> (GLfloat, GLfloat) {
```

Comment:
Why the name, instead of `aliased_line_width_range`?

Reply:
The constant was wrong and the method name ok.